### PR TITLE
Timeout waiting for connection from pool (again)...

### PR DESCRIPTION
**Pull request by Marczeeee**
Issue #202

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, I'll find someone to review this PR soon

**Comment by davvd** [comment]
@krzyk this pull request is for you, please review

File: s3auth-hosts/src/test/java/com/s3auth/hosts/DefaultResourceTest.java [hunk 1]
```diff
@@ -367,4 +368,37 @@ public void getsContentEncodingHeaderFromAmazonObject() throws Exception {
         );
     }
 
+    /**
+     * Reproduces the scenario when an Amazon S3 bucket connection throws
```
Comment (krzyk) [inline s3auth-hosts/src/test/java/com/s3auth/hosts/DefaultResourceTest.java]: please see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names

**Comment by krzyk** [comment]
@Marczeeee I think this doesn't solve the problem, it will be just logged earlier

**Comment by davvd** [comment]
@krzyk **18 mins** sent to your balance (ID `50525528`), many thanks!. added +18 to your rating, now it is equal to [+2086](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**